style.css

```diff
@@ -0,0 +1,433 @@
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: url(https://images.pexels.com/photos/1109543/pexels-photo-1109543.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+    filter: brightness(65%);
+}
+
+.hero-content {
+    position: relative;
+    text-align: center;
+    color: #fbf0f0;
+    margin: 0.625rem;
+}
+
+.hero-title {
+    font-size: 3rem;
+    font-weight: 600;
+    margin-bottom: 0;
+    Font-Family: 'Roboto Condensed', Sans-Serif;
+    Font-Size: 60px;
+    letter-spacing: 3px;
+}
+
+.hero-subtitle {
+    font-size: 2rem;
+    font-weight: 200;
+    margin-top: 1rem;
+    Font-Family: 'Cabin', Sans-Serif;
+    Font-Size: 30px;
+    letter-spacing: 3px;
+}
+
+.hero-button {
+   border: none;
+   margin-top: 2rem;
+   padding: 20px;
+   border-radius: 5px;
+   font-size: 1.125rem;
+   font-weight: 200;
+   cursor: pointer;
+   background: #fbf0f0;
+   box-shadow: 0px 3px 2px 0 #fbf0f0;
+   Font-Family: 'Cabin', Sans-Serif;
+   Font-Size: 15px;
+   letter-spacing: 3px;
+}
+
+.hero-button:hover {
+   background-color: #fbf0f0;
+   box-shadow: none;
+   border: none;
+   transform: translateY(5px);
+}
+
+/*** Main section with accordion and form styling ***/
+
+main{ 
+    background: #dfd3d3;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+/*** Accordion styling ***/
+
+.accordion-container {
+    margin-top: 2rem;
+    border: 1px solid #b8b0b0;
+    border-radius: 5px;
+    margin-bottom: 15px;
+    width: 500px;
+    background: #fbf0f0;
+    padding: 15px;
+}
+
+.accordion-header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding: 15px;
+    cursor: pointer;
+    Font-Family: 'Roboto Condensed', Sans-Serif;
+    letter-spacing: 3px;
+}
+
+.accordion-text {
+    border-top: 2px solid #7c7575;
+    margin: 0;
+    padding: 15px;
+    Font-Family: 'Roboto Condensed', Sans-Serif;
+}
+
+.accordion-text-container {
+    height: 0;
+    overflow: hidden;
+    transition: .3s se;
+}
+
+.accordion-text-container.active {
+    height: 158px;
+}
+
+/*** Form styling ***/
+
+fieldset{
+    width: 100%;
+    border: none;
+}
+.form-container {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+    width: 500px;
+    background: #fbf0f0;
+    padding: 15px;
+    border: 1px solid #b8b0b0;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+    Font-Family: 'Roboto Condensed', Sans-Serif;
+}
+
+.form-group {
+    margin-bottom: 5px;
+}
+
+.form-group label {
+    width: 100px;
+    padding: 10px;
+    Font-Family: 'Roboto Condensed', Sans-Serif;
+}
+
+.form-group input,
+.form-group textarea,
+.form-group select {
+    width: 250px;
+    padding: 10px;
+    border: none;
+    background:#fbf0f0;
+    display: flex;
+    Font-Family: 'Cabin', Sans-Serif;
+    Font-Size: 15px;
+}
```

Comment:
You form looks really cool and sleek, however it's hard to realise that it actually is a form since it blends into the background. I think you should give the background another color or at least a border for it to be visual. We got to think about people with vision problem too when we create a website - so something to keep in mind!